packages/ncids-css/scss/components/nci-header/internal/_navigation-mobile.scss

```diff
@@ -0,0 +1,206 @@
+  &.active {
+    display: flex;
+  }
+  &
+    .nci-header--nci-mobilenav
+    > .nci-header--nci-mobilenav__node:first-child:not(.active) {
+    border-top: 1px solid color("gray-warm-10");
+    margin-top: units(6);
+  }
+}
+
+.nci-mobile-overlay {
+  &.active {
+    display: block;
+  }
+  display: none;
+
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  content: "";
+  background: rgba(0, 0, 0, 0.2);
+}
+
+.nci-header--nci-mobilenav {
+  font-size: units(2);
+  list-style-type: none;
+  line-height: 1.3;
+  max-width: 430px;
+  margin: 0;
+  padding-left: 0;
+
+  &__item {
+    margin-bottom: 0;
+    max-width: unset;
+    &:not(last-child) {
+      border-bottom: 1px solid color("gray-warm-10");
+    }
+  }
+
+  &__link {
+    color: color("gray-warm-60");
+    display: block;
+    padding: units(1) units(2);
+    position: relative;
+    text-decoration: none;
+    &:hover {
+      cursor: pointer;
+      background: color("gray-warm-2");
+      color: color("gray-warm-60");
+    }
+    &.hover {
+      cursor: pointer;
+      background: color("gray-warm-2");
+      color: color("gray-warm-60");
+    }
+    &.current::after {
+      background-color: color("primary");
+      border-radius: 99rem;
+      content: "";
+      display: block;
+      position: absolute;
+      bottom: 0.25rem;
+      top: 0.25rem;
+      width: 0.25rem;
+      left: 0.25rem;
+    }
+  }
+
+  &__node {
+    border-bottom: 1px solid color("gray-warm-10");
+    color: color("gray-50");
+    padding: units(1) units(2);
+    text-decoration: none;
+    &:hover {
+      background-color: color("gray-warm-2");
+      color: color("gray-50");
+      cursor: pointer;
+      text-decoration: none;
+    }
+    &.hover {
+      background-color: color("gray-warm-2");
+      color: color("gray-50");
+      cursor: pointer;
+      text-decoration: none;
+    }
+    #nci-mobilenav__msg {
+      display: none;
+      @include add-background-svg("usa-icons/chevron_left");
+      background-size: units(3) units(3);
+      background-repeat: no-repeat;
+      background-position: -8px center;
+    }
+    #nci-mobilenav__label {
+      display: block;
+      @include add-background-svg("usa-icons/chevron_right");
+      background-size: units(3) units(3);
+      background-repeat: no-repeat;
+      background-position: right center;
+
+      padding-right: units(2);
+    }
+    &.active {
+      color: color("primary");
+      font-weight: 700;
+
+      padding-bottom: units(2.5);
+      padding-left: 0px;
+
+      & > #nci-mobilenav__msg {
+        display: block !important;
+        padding-left: units(2);
+      }
+      & > #nci-mobilenav__label {
+        display: none !important;
+        padding-left: 0;
+      }
+      &:hover {
+        background: transparent;
+      }
+    }
+  }
+
+  &__holder {
+    padding: 0px 0px;
+  }
+}
+
+.nci-header--nci-mobilenav > li > a {
+  padding: units(1) 4px units(1) units(2);
+}
+.nci-header--nci-mobilenav
```

Comment:
this still looks like an overly complicated selector.
Would the following not work?
```
.nci-header--nci-mobilenav__node:not(.active),
.nci-header--nci-mobilenav__item:not(:first-child) {
  padding: units(1) 4px units(1) units(3);
}

Reply:
no because of the specificity of the nested UL's in that area
![Screen Shot 2022-03-28 at 10 40 51 AM](https://user-images.githubusercontent.com/66697447/160423699-fd07ea39-c3da-4b3c-b171-b1ffb2cdaa8f.png)